prisma/seed.ts

```diff
@@ -68,25 +67,35 @@ async function main() {
     const organization = seedData.at(i);
 
     if (organization) {
-      const shortDescription =
-        organization.description.length > 200
-          ? organization.description.slice(0, 200).trim() + "..."
-          : organization.description;
-
+      console.log(organization.name);
       const logo = findLogo(organization);
-
       if (logo) {
         addedLogos.push(logo);
       }
+      const contactMethods = [];
+      if (organization.contact) {
+        for (const [contactType, contactLink] of Object.entries(
+          organization.contact
+        )) {
+          contactMethods.push({
+            contactType,
+            contactLink: contactLink as string,
```

Comment:
dlaczego tu jest potrzebne castowanie

Reply:
Bez tego castowania krzyczy na mnie eslint, nawet gdy otypowałem obiekt contactMethods to nadal pojawia się ten błąd
![image](https://github.com/informatyzacja/strona-organizacji-studenckich/assets/61631855/5ba0568a-7e32-4966-b354-ff6c8fa30f5b)

Reply:
weird, oki